src/components/GoogleSignIn/GoogleSignIn.tsx

```diff
@@ -24,6 +24,8 @@ function GoogleSignIn(props: Props) {
         setGoogleId(response.tokenId);
         api.setGoogle(response.tokenId).then(() => {
             props.onAfterLogin();
+        }).catch(() => {
+            googleId = null;
```

Comment:
this doesn't fix #71
I think, it has to at least delete the token from localstorage as well